pkg/app/app.go

```diff
@@ -38,23 +39,25 @@ func initConfiguration(config configuration.Configuration) {
 		}
 	})
 
-	config.AddDefaultValue(configuration.ORGANIZATION, func(existingValue interface{}) interface{} {
-		useDefaultValue := true
-		orgid := existingValue
-		if existingValue != nil {
-			if len(existingValue.(string)) > 0 {
-				useDefaultValue = false
+	config.AddDefaultValue(configuration.ORGANIZATION, func(existingValue any) any {
+		client := networking.NewNetworkAccess(config).GetHttpClient()
+		apiClient := networking.NewApi(configuration.API_URL, client)
```

Comment:
chore: `configuration.API_URL` is a key and must actually be used like this `config.GetString(configuration.API_URL)` to actually retrieve the URL, which is a configurable value. See also https://github.com/snyk/go-application-framework/pull/15/files#diff-63facfa8b028c9f397524787fac074737e5534ae2591640c1a029549ec3d7450L51

Reply:
Thanks! I'll fix this